src/pages/coordinator/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { useCallback } from "react";
+
+import { Layout } from "~/layouts/DefaultLayout";
+import { useIsAdmin } from "~/hooks/useIsAdmin";
+import { useDeployment } from "~/hooks/useDeployment";
+
+
+export default function CoodirnatorDashboard() { 
+    const isAdmin = useIsAdmin();
```

Comment:
should we also check if there's already data in localstorage for the contracts and resume from there in case? so set the step to the latest step (maybe we should be storing the step in the json too)

Reply:
ok actually I see it does that, but it doesn't allow to go back in case there was an error at any time

Reply:
it's actually doing this now, if there are stored contract in the localStorage, it will skip that step.
But this is not reflected on the UI now.

Maybe it would be good to add a flag called `forceToDeploy` in the MaciService functions, in case user wanna deploy regardless of the contract in the localStorage.

Reply:
yea that might be useful

Reply:
resolve this conversation first since wanna open another issue and PR for this.